### PR TITLE
Adding some links and clarifications to the contributing new documentation section

### PR DESCRIPTION
**Pull request by SylLutze**

### CODE REVIEW

File: docs/source/onboarding_resources/contributing/index.rst [hunk 1]
```diff
@@ -7,30 +7,42 @@ Contributing New Documentation
...
+This document is designed to guide someone through their first time contributing 
+documentation to Vivarium Research. This process can be confusing though, so please 
+ask a friend on the team for help if needed or to clarify any questions!! 
+
 Useful links
 ------------
 
-Our documentation is written using reStructuredText and Sphinx. The following
+Our documentation is written using reStructuredText and Sphinx. These are ways of writing 
+text to include formatting and structure to documents. The following
```
Comment (zmbc) [inline docs/source/onboarding_resources/contributing/index.rst:18]:
```suggestion
text to include formatting and structure in documents. The following
```

File: docs/source/onboarding_resources/contributing/index.rst [hunk 2]
```diff
@@ -7,30 +7,42 @@ Contributing New Documentation
...
 
+Start by opening the terminal of your choosing. If you want more information on 
+this, check out this :ref:`page on terminals <terminal_access>`. Then enter 
+the commands below. 
+
 ::
 
-   $> conda create -y --name=vivarium_research python=3.6
+   $> conda create -y --name=vivarium_research python=3.8
    $> source activate vivarium_research
```
Comment (zmbc) [inline docs/source/onboarding_resources/contributing/index.rst:38]:
```suggestion
   $> conda activate vivarium_research
```

I think this is the recommended way these days.

File: docs/source/onboarding_resources/contributing/index.rst [hunk 3]
```diff
@@ -43,25 +55,44 @@ The ``YOUR-BRANCH-NAME`` is the name of your branch, and you should try to
...
+   (base) $> conda activate vivarium_research ##This line is only needed if vivarium_research has not been activated yet
    (vivarium_research) $> make html
 
 This will create a new ``build`` sub-directory with the new documentation
-rendered in html.  You can open the file `docs/build/html/index.html` in your
-browser to view your changes.
+rendered in html.  You can open the file `vivarium_research/docs/build/html/index.html` in your
+browser to view your changes. 
+
+Note that some errors will appear as warnings in Miniconda, but will actually cause the build to
```
Comment (zmbc) [inline docs/source/onboarding_resources/contributing/index.rst:93]:
```suggestion
Note that any warnings in your terminal after the `make html` command will cause the build to 
```

This isn't a difference between local and GitHub per se, it's just that we have set up GitHub to fail on any warnings.

**Review by zmbc — COMMENTED** [review]
I am concerned that this page is redundant with [the section of the README that aims to explain the same thing](https://github.com/ihmeuw/vivarium_research#id1).

I'd propose:
- Merge all relevant content from the README into this document.
- Delete the content in the README and replace it with a link to this page.

**Review by aflaxman — APPROVED** [review]

**Review by alibow — APPROVED** [review]
Thanks for adding all this detail! While we're at it, I wonder if we should also add the following:

- A link to resources on git/best practices (IHME trainings?)
- A note about deleting branches that have been merged (both on github as well as locally). While this isn't really necessary, it's nice just to keep things tidy.

**Pull request merged**